packages/ui-app/src/CardSummary.tsx

```diff
@@ -90,7 +105,7 @@ export default class CardSummary extends React.PureComponent<Props> {
     }
 
     return (
-      <Card>
+      <Card className={className}>
```

Comment:
Would be useful to pull style in as well (prefer not using them, but it just removed confusion if all components work with them without problems)